osu.Game.Rulesets.Osu/Mods/OsuModRelax.cs

```diff
@@ -17,46 +18,49 @@
 
 namespace osu.Game.Rulesets.Osu.Mods
 {
-    public class OsuModRelax : ModRelax, IUpdatableByPlayfield, IApplicableToDrawableRuleset<OsuHitObject>, IApplicableToPlayer
```

Comment:
This is ugly but without multiple inheritance it was the best I could come up with. Definitely open to suggestions.